scripts/variantstore/wdl/GvsPrepareBulkImport.wdl

```diff
@@ -45,9 +48,10 @@ task GenerateFOFNsFromDataTables {
         String? sample_id_column_name
         String? vcf_files_column_name
         String? vcf_index_files_column_name
+        String? sample_set_name
     }
 
-    ## TODO I dont love that we are hardcoding them here and in the python--they need to be params!
+    ## TODO these are now in the python script, but should they no longer be here?
     String sample_names_file_name = "sample_names.txt"
     String vcf_files_name = "vcf_files.txt"
     String vcf_index_files_name = "vcf_index_files.txt"
```

Comment:
reviewers: is it overkill to have these values here (and passed to the python script) if they are ALSO in the python script as default values and they really are just hard coded through and through

Reply:
Duplication and hardcoding-in-disguise certainly doesn't sound good... 🙂